jsonnet/kube-prometheus/lib/prometheus.libsonnet

```diff
@@ -0,0 +1,7 @@
+{
+  // rangeInterval takes a scrape interval and convert its to a range interval
+  // following Prometheus rule of thumb for rate() and irate().
+  rangeInterval(i='1m'):
+    local interval = std.parseInt(std.substr(i, 0, std.length(i) - 1));
+    interval * 4 + i[std.length(i) - 1],
```

Comment:
I am not sure we need this as an external lib. Maybe let's move it to `main.libsonnet` close to its usage?

Reply:
Thinking a bit more, what if we simplified it to a function like `getScrapeInterval` that would return a number of seconds for a particular scrape interval? This way we could do sth like:
```
rangeInterval: {
  kubelet: utils.getScrapeInterval($.kubernetesControlPlane.serviceMonitorKubelet.spec.endpoints[0].interval) * 4,
}
```

We could put it into sth like `jsonnet/kube-prometheus/lib/utils.libsonnet` and start small utulity-related lib with such snippets. WDYT?

Reply:
I am not a fan of making `main.libsonnet` more complex. To me, it should be as minimal as possible to improve code readability. That said, I went with your other suggestion of renaming the lib to `utils`.